assets/ajax-cart.js.liquid

```diff
@@ -92,40 +106,55 @@ ShopifyAPI.addItemFromForm = function(form, callback, errorCallback) {
       else {
         ShopifyAPI.onError(XMLHttpRequest, textStatus);
       }
+      $body.trigger('errorAddItem.ajaxCart', [XMLHttpRequest, textStatus]);
+    },
+    complete: function(jqxhr, text) {
+      $body.trigger('completeAddItem.ajaxCart', [this, jqxhr, text]);
     }
   };
   jQuery.ajax(params);
 };
 
 // Get from cart.js returns the cart in JSON
 ShopifyAPI.getCart = function(callback) {
+  $(document.body).trigger('beforeGetCart.ajaxCart');
   jQuery.getJSON('/cart.js', function (cart, textStatus) {
     if ((typeof callback) === 'function') {
       callback(cart);
     }
     else {
       ShopifyAPI.onCartUpdate(cart);
     }
+    $(document.body).trigger('afterGotCart.ajaxCart', cart);
```

Comment:
Last change, switch this to `afterGetCart`. Realizing the naming issues as I'm writing up some new docs